custom_components/tuya_local/devices/prosto_ptc2000_heater.yaml

```diff
@@ -0,0 +1,91 @@
+name: Prosto Heater
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: preset_mode
+          conditions:
+            - dps_val: level_0
+              value: fan_only
+            - dps_val: level_1
+              value: heat
+            - dps_val: level_2
+              value: heat
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 18
+        max: 45
+    - id: 3
+      type: integer
+      name: current_temperature
+    - id: 5
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: "level_0"
+          value: fan
+        - dps_val: "level_1"
+          value: comfort
+        - dps_val: "level_2"
+          value: boost
+    - id: 4
```

Comment:
level is not part of the climate API, so this probably needs to be a separate select entity.